### DISCUSSION TITLE
What is the FlutterSDK version you are using for the source you are developing?

### DISCUSSION THREAD

**Discussion by andrwj** [imrofayel/Ozan#28] [Q&A] [answered]
I've tried building with different versions of the FlutterSDK, but they all fail.

**Comment by imrofayel** [comment, marked as answer]
from `flutter --version` I guess I didn't upgrade the SDK for a while now.

```terminal
Flutter 3.19.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision bae5e49bc2 (8 months ago) • 2024-02-13 17:46:18 -0800
Engine • revision 04817c99c9
Tools • Dart 3.3.0 • DevTools 2.31.1
```

It's sad to hear this is happening. I hope it gets resolved soon, and you're able to contribute to the project! This was my first project ever so the code is little weird tbh, `if it works, dont touch it` strategy haha.

Have a great day!
